src/timeout.js

```diff
@@ -19,8 +20,7 @@ class Timeout {
 
 	rejected (p) {
 		clearTimeout(this.timer)
-		this.promise._become(p)
-		return false
+		return super.rejected(p)
```

Comment:
It doesn't matter too much to me, honestly.  It was just a suggestion for consistency.